### PR TITLE
Cleanup builder when we are done.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -105,6 +105,13 @@ jobs:
...
     if: github.event_name == 'release'
     steps:
+      - name: Clean workspace at the end
+        uses: webiny/action-post-run@3.0.0
+        id: post-run-command
+        with:
+          working-directory: ${{ github.workspace }}/..
+          run: rm -rf ${{ github.workspace }}
+
       - uses: actions/checkout@v3
```
Comment (kasperl) [inline .github/workflows/build.yml:115]: Do we need checkout here? You removed it elsewhere.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/build.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/build.yml:112]: Yes, because we are creating an installer which needs files from the repository.

**Comment by floitsch** [comment]
PTAL: the 'rm' didn't work correctly on Windows.

Note that this approach is actually nicer, since it leaves the files at the end, which allows us to debug them more easily. Only when starting a new job do we clean the directory.

**Review by kasperl — APPROVED** [review]
Looks good to me.

**Pull request merged**